src/tools/rustc-perf-wrapper/src/config.rs

```diff
@@ -1,5 +1,7 @@
 use std::fmt::{Display, Formatter};
 
+pub const DB_PATH: &str = "./build/results.db";
```

Comment:
The directory name is not always `build`, it should be taken from `args.ctx.results_dir`.